### PR TITLE
README: use rootfs image; more and better podman

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,13 @@ line.
 
 ## Installing from ISO
 
-[Download a Fedora CoreOS ISO image](https://getfedora.org/coreos/download/).
+Download a Fedora CoreOS ISO image:
+
+```
+podman run --privileged --pull=always --rm -v .:/data -w /data \
```
Comment (lucab) [inline README.md:125]: Nit: is `--privileged` really needed for the `download` verb?

File: README.md [hunk 2]
```diff
@@ -160,7 +166,13 @@ embedded Ignition config will run on first boot.
 
 ## Installing from PXE
 
-[Download a Fedora CoreOS PXE kernel and initramfs image](https://getfedora.org/coreos/download/).
+Download a Fedora CoreOS PXE kernel, initramfs, and rootfs image:
+
+```
+podman run --privileged --pull=always --rm -v .:/data -w /data \
```
Comment (lucab) [inline README.md:172]: Nit: is `--privileged` really needed for the `download` verb?

**Comment by lucab** [comment]
LGTM. I have two small questions/nits, but feel free to merge otherwise.

File: README.md (hunk 1 above)
Comment (bgilbert) [reply README.md:125]: Assuming `.` is under `$HOME`, SELinux will deny the write without it.

**Pull request merged**